internal/conduit/registry/geth_block_test.go

```diff
@@ -42,6 +43,7 @@ func (ec *EthClientMocked) BlockByNumber(ctx context.Context, number *big.Int) (
 func Test_ConfigureRoutine_Error(t *testing.T) {
 
 	ctx, cancel := context.WithCancel(context.Background())
+	logging.NewLogger(nil, false)
```

Comment:
It may be easier to just have a function called `NewNoopLogger` that uses the existing zap logic: https://pkg.go.dev/go.uber.org/zap#NewNop.